src/features/chart/components/Chart.js

```diff
@@ -11,9 +11,9 @@ import {
 } from "recharts";
 // import moment from 'moment';
 import { useSelector, useDispatch } from "react-redux";
-import { updateChartAction } from "../../store/actions";
-import { Loading } from "../loading";
-import { ErrorMessage } from "../error-message";
+import { updateChartAction } from "../redux/actions";
+import { Loading } from "../../loading/components/Loading";
```

Comment:
nit: This import shouldn't have to change. The component should be exported from an `index.js` file in the `loading` feature folder.

Same applies to every other feature folder, by the way. You should consider what a feature's "API" consists of:
1. Component
2. Reducer
3. Middleware
4. Actions

Reply:
We'll change this in another refactor branch